hw/ip/rv_dm/lint/rv_dm.waiver

```diff
@@ -12,12 +12,24 @@ waive -rules IFDEF_CODE -location {rv_dm.sv} -regexp {DMIDirectTAP} \
 waive -rules CLOCK_MUX  -location {rv_dm.sv} \
       -regexp {Clock 'tck_muxed' is driven by a multiplexer here, used as a clock} \
       -comment "The signal is coming out of a prim_clock_mux2 block."
+waive -rules CLOCK_USE -location {rv_dm.sv} \
+      -msg {'clk_i' is connected to 'prim_clock_mux2' port 'clk1_i',} \
+      -comment "This is needed such that the clock can be properly controlled during scanmode."
+
 
 # dmi_jtag_tap
 waive -rules CLOCK_MUX -location {dmi_jtag_tap.sv} -regexp {Clock 'tck_n' is driven by a multiplexer here, used as a clock at dmi_jtag_tap.sv} \
       -comment "This is permissible for now, but should be revised as soon as ASIC targets come into play."
 waive -rules INV_CLOCK -location {dmi_jtag_tap.sv} -regexp {'tck_i' is inverted, used as clock at dmi_jtag_tap.sv} \
       -comment "This is permissible for now, but should be revised as soon as ASIC targets come into play."
+waive -rules CLOCK_DRIVER -location {dmi_jtag_tap.sv} -msg {'tck_n' is driven by instance 'i_tck_inv' of module 'prim_clock_inv'} \
+      -comment "This is due to clocking primitives on the clock line."
+waive -rules CLOCK_DRIVER -location {dmi_jtag_tap.sv} -msg {'clk_no' driven in module 'prim_clock_inv' by port } \
+      -comment "This is due to clocking primitives on the clock line."
+waive -rules CLOCK_DRIVER -location {dmi_jtag_tap.sv} -regexp {'clk_no' driven in module.*} \
+      -comment "This is due to clocking primitives on the clock line."
```

Comment:
Maybe it makes sense for these two rules to contain more of the message/regexp (btw. what's the difference?) to be less broad. For example, include the port name etc. This way the lint tool might cause false alarm when we change something. We then need to adjust the waiver which is easy. What I am worried about is that we change something on these clocks the wrong way round and the lint tool doesn't notice it because the rule is too broadly scoped.

Reply:
I'll try to make them a bit more specific